R/module_nested_tabs.R

```diff
@@ -68,16 +68,24 @@ ui_teal_module.teal_modules <- function(id, modules, depth = 0L) {
       lapply(
         names(modules$children),
         function(module_id) {
-          module_label <- modules$children[[module_id]]$label
-          tabPanel(
-            title = module_label,
-            value = module_id, # when clicked this tab value changes input$<tabset panel id>
-            ui_teal_module(
-              id = ns(module_id),
-              modules = modules$children[[module_id]],
-              depth = depth + 1L
+          if (module_id == "teal_data_module") {
+            tabPanel(
```

Comment:
@vedhav 
Looks for me like `ui_teal_module.shiny.tag` instead of making this if trick here. Or make pseudo `teal_module` in `ui_teal` so that it can be handled without changes in `ui_nested_tabs`